modules/300-prometheus/docs/FAQ_RU.md

```diff
@@ -651,4 +651,46 @@ spec:
   metricsPath: '/metrics'
 ```
 
+## Как опрашивать дополнительные эндпоинты используя Probe?
```

Comment:
Let's change to:
```
Как настроить Probe для работы с Prometheus?
```

Reply:
Additionally, can you move this section closer to the Service Monitor section, placing it under that section?

Reply:
fixed